src/main/java/com/nukkitx/proxypass/Configuration.java

```diff
@@ -26,6 +26,8 @@
     private boolean packetTesting = false;
     @JsonProperty("log-packets")
     private boolean loggingPackets = false;
+    @JsonProperty("log-to")
+    private String logTo = "file";
```

Comment:
change this to an enum with the values FILE, CONSOLE and BOTH

Reply:
Done